mkdocs.yml

```diff
@@ -69,9 +69,16 @@ nav:
     - Metadata: 'icommands/metadata.md'
     - Tickets: 'icommands/tickets.md'
     - Groupadmin: 'icommands/groupadmin.md'
+ - Policy Cookbook:
+    - Home: 'policy_cookbook/index.md'
```

Comment:
not sure about 'Home'... but don't have a better suggestion at the moment...

Reply:
Same here

Reply:
Here are some suggestions, which may or may not be better...
 - Index
 - Table of Contents
 - Overview
 - Introduction

Alternatively, this could all go under one page (much like the library examples or troubleshooting pages) and stuck under System Overview or even grow another section for administrators.

Reply:
Updated navigation.

Added section called `Administrators` and placed the `Policy Cookbook` under that.

`System Overview` could work, but the word, `Overview`, doesn't fit the information IMO.

Thoughts?

Reply:
@trel @alanking Any thoughts on the navigation?

You can see the current rendering by going to kdd-ws.euro...org:8080

Reply:
i like the navigation so far.   I think it's in the right place for now.

Reply:
Excellent.